test/Performance/HelloWorld.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
We could.  However, I was envisioning this test eventually doing more.  Today, we are most concerned with build, but I could imagine us wanting to expand the number of operations, at which point the name of the file would change too.